docs/reference/remap/functions/is_log.cue

```diff
@@ -0,0 +1,26 @@
+package metadata
+
+remap: functions: is_log: {
+	arguments: [
+	]
+	internal_failure_reasons: []
+	return: {
+		types: ["boolean"]
+		rules: [
+			#"If the current event is a log event, then `true` is returned."#,
+		]
+	}
+	category: "Type"
+	description: #"""
+		Determines whether the current event is a log event.
+		"""#
+	examples: [
+		{
+			title: "A log event"
+			source: """
+				is_log()
```

Comment:
I think it'd be best to provide more realistic, even if contrived, examples for these two functions. Maybe something an `if` statement?

Reply:
I'm not quite sure what I could do. These functions are really designed for use mainly with the conditions, so the main source probably would be just `is_log()`, or perhaps `is_log() && contains(.message, "turnips")` - but I'm not sure if that makes for a good example..

Reply:
Yes, we should support an `input` with a log event for this example, since it operates directly on the event. Ex:

```
input: log: message: "Hello, world!"
```